CONTRIBUTING.md

```diff
@@ -86,6 +87,33 @@ is not a problem to open a merge request for incomplete implementations.
 10. Your merge request can now be merged in. Congratulations and thank you so much, you have contributed something new to
     the repository.
 
+## Submitting a pull request
+
+As soon as there exists something in your branch, a pull request can be opened on the main repository. Do not forget that it
+is not a problem to open a pull request for incomplete implementations.
+
+1. Retrieve the latest changes from the upstream version as explained above.
+2. Optionally format the code if you did not add the git-hook from the beginning, this can be done manually by running
+   `make format` from the build directory.
+3. Go to [pull request](https://github.com/your-username/allpix-squared) and click on "Pull requests" and then "New pull
+request".
+4. Follow the instructions. Do not forget to select the "Create draft pull request" if your code is only partially ready.
+Then submit the pull request.
+5. Please wait for the maintainers to give you access to the continuous integration (CI) runners that will check your code
```

Comment:
This one can go out I think, GitHub actions will run automatically, and we soon also have an auto-mirror to GitLab. :)

Reply:
Similarly, the remaining CI-related items do not apply for GitHub